etc/profile-a-l/kcalc.profile

```diff
@@ -41,13 +47,19 @@ nou2f
 novideo
 protocol unix
 seccomp
+seccomp.block-secondary
 shell none
+tracelog
 
 disable-mnt
 private-bin kcalc
+private-cache
 private-dev
+private-etc alternatives,fonts,ld.so.cache,locale,locale.conf
 # private-lib - problems on Arch
 private-tmp
 
 dbus-user none
 dbus-system none
+
+#memory-deny-write-execute
```

Comment:
If mdwe breaks it does not need to be added.

Reply:
on arch it works smoothly, have to test on other distros.